### PR TITLE
Updated Trezor plugin to trezor>=0.9.0

### PR DESCRIPTION
**Pull request by jhoenicke**
There have been some backward incompatible changes to python-trezor.  With these patches electron-cash runs with the latest version of python-trezor.

### CODE REVIEW

**Comment by jhoenicke** [comment]
See also https://github.com/fyookball/electrum/issues/417

**Comment by marceloneil** [comment]
This isn't electron-gold, doesn't electron-gold steal private keys? #280

File: plugins/trezor/trezor.py [hunk 1]
```diff
@@ -17,23 +17,20 @@ def __init__(self, *args):
...
-        return HidTransport(pair)
+        return HidTransport.find_by_path(device.path)
 
     def bridge_transport(self, d):
         from trezorlib.transport_bridge import BridgeTransport
         return BridgeTransport(d)
-
-    def get_coin_name(self):
-        # No testnet support yet
-        return "Bcash"
```
Comment (kyuupichan) [inline plugins/trezor/trezor.py]: Uhh where did get_coin_name go?

**Comment by kyuupichan** [comment]
This seems to remove get_coin_name and not provide any replacement?

File: plugins/trezor/trezor.py (hunk 1 above)
Comment (jhoenicke) [reply plugins/trezor/trezor.py]: It moved to the base class in plugin.py.  I moved it to make it more similar to the electrum code base.

Comment (kyuupichan) [reply plugins/trezor/trezor.py]: Ah sorry, missed that.

**Comment by SomberNight** [comment]
Don't you also need to backport https://github.com/spesmilo/electrum/commit/cc9032c9ea79a2f44e0b251699838102f7181b66 ?

**Pull request merged**